src/errorCodes.js

```diff
@@ -17,7 +17,9 @@ const codes = {
     notAt: 'notAt',
     notADate: 'notADate',
     invalidFormat: 'invalidFormat',
-    invalidURL: 'invalidURL'
+    invalidURL: 'invalidURL',
+    invalidInclusion: 'notContains',
```

Comment:
```
    notContains: 'notContains',
    contains: 'contains'
```

Reply:
I'll work on this sugestions and return asap. 
I am very happy with the progress of this feature.